src/xdocs/config_sizes.xml

```diff
@@ -524,6 +524,20 @@ class Test {
               <td><code>80</code></td>
               <td>3.0</td>
             </tr>
+            <tr>
+              <td>validateImport</td>
+              <td>Validate checking of import statements.</td>
+              <td><a href="property_types.html#boolean">boolean</a></td>
+              <td><code>false</code></td>
+              <td>8.45</td>
```

Comment:
should be 8.46 , we did release few days ago.

Next release will be last weekend of August, please try to find time to fix it before.